src/rard/utils/basemodel.py

```diff
@@ -303,6 +303,12 @@ def link_bibliography_mentions(self):
                 if self.book:
                     antiquarians = {ant for ant in self.book.work.antiquarian_set.all()}
 
+                if self.citing_work:
+                    antiquarians = {self.citing_work.author}
```

Comment:
This works, but I think it's cheating. Since this variable no longer necessarily refers exclusively to Antiquarian model instances, I think it needs renaming to something like `bibliography_owners`.